docs/setting-up-fuzzing/libfuzzer_and_afl.md

```diff
@@ -100,23 +100,23 @@ To create a job for libFuzzer or AFL:
 1. Navigate to the *Jobs* page.
 2. Go to the "ADD NEW JOB" form.
 3. Fill out the "Name" and "Platform" (LINUX).
-  1. If setting up an **AFL** job, use the templates **"afl"** and
-     **"engine_asan"**.
-  2. If setting up a **libFuzzer** job, use the templates **"libfuzzer"** and
-     **"engine_$SANITIZER"** depending on which sanitizer you are using (e.g.
-     **"engine_asan"**).
-4. Select your build (your zip containing the fuzz target binary) to upload as a
+4. Enable the desired fuzzer in the "Select/modify fuzzers" field, e.g. **libFuzzer** or **afl**.
+5. If setting up an **AFL** job, use the templates **"afl"** and
```

Comment:
not sure why we had the indentation here, it is rendered as a straight list anyway: https://google.github.io/clusterfuzz/setting-up-fuzzing/libfuzzer-and-afl/#creating-a-job-type